crates/lib-dialects/test/fixtures/dialects/tsql/alias_equals_syntax.yml

```diff
@@ -0,0 +1,29 @@
+file:
+- statement:
+  - select_statement:
+    - select_clause:
+      - keyword: SELECT
+      - select_clause_element:
+        - expression:
+          - column_reference:
+            - naked_identifier: TsqlAlias
+          - comparison_operator:
+            - raw_comparison_operator: =
```

Comment:
This isn't quite synactically correct. I think we need to somehow represent this in an alias.

Reply:
Changed https://github.com/quarylabs/sqruff/pull/1664/commits/4e0bd90ea8801927d780441381e35cdf6ea08b1c